ppl/src/main/antlr/OpenSearchPPLLexer.g4

```diff
@@ -95,11 +95,18 @@ INTERVAL:                           'INTERVAL';
 MICROSECOND:                        'MICROSECOND';
 MILLISECOND:                        'MILLISECOND';
 SECOND:                             'SECOND';
+SECOND_OF_MINUTE:                   'SECOND_OF_MINUTE';
```

Comment:
Can you groom this list to be in alphabetical order?

Reply:
fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/281/commits/7b927df652fd09ef0e74315eb6a2abfbffe1f8f3